share/spice/sublime_packages/footer.handlebars

```diff
@@ -1,4 +1,10 @@
 <div class="tile__update__info one-line">
-	<i class="ddgsi ddgsi-arrow-down"></i>
-	{{unique_installs}} &middot; Updated {{last_modified}}
+        <span class="opt--left">
```

Comment:
@cinlloc It looks like our `opt--left` and `opt--right` classes aren't working right now. I'll switch these to `pull-left`, `pull-right` which do work.